source/support/data-protection/index.html.md.erb

```diff
@@ -0,0 +1,162 @@
+
+This guide is not an complete list of everything we must do under the UK GDPR. Instead it focuses on user rights
+requests and how to respond.
+
+# Who handles requests?
+
+Responding to rights requests is a collaboration between the Privacy Office, User support team and GOV.UK Accounts team.
+All requests should initially arrive through zendesk where they can be flagged by User Support.
+
+If a request arrives directly with the accounts team they should create a new zendesk ticket. Where possible the ticket should contain:
+- Contact details for the user who has made the request
+- What their request is
+- When their request arrived with the GOV.UK account team
+- @TODO (any tags?)
+
+It can then be assigned to @TODO (???)
+
+Privacy office will work with user support to ensure the request is valid. If the request requires further actions from
+the accounts team user support will @TODO (How will they make contact?)
+
+# Time limits
+
+Collectively GOV.UK will have 30 calendar days to respond to any rights request. The clock does not stop for non-working days such as bank holidays or weekend.
+
+Where the Accounts team receives a request we should assign it to user support as soon as we see it.
+
+If user support asks accounts to provide information or take an action as part of a rights request, we should respond to user support within 7 days.
+
+# Self service
+
+The GOV.UK Account is designed to give users the maximum possible control over there data, so users will often be able to self serve to exercise their rights. See the [Prepared responses to common issues][1] page for messages that help users find the right part of the account.
+
+If the user is unwilling to use their account, or there are elements to their request they cannot achieve themselves, the request should be brought into the team, put on a trello ticket and assigned to a developer.
+
+# Types of requests
+
+There are different types of requests we may receive. The section below describes a user's rights and actions we may have to take in response to them.
+
+## Requests to access data
+
+Also been known as Subject Access Requests. A user with an account has a right to see information about them held on the system. This can help a user better understand how their data is being use, and provide a way check we are
+meeting promises we made in the [govuk-accounts-privacy-policy][2].
+
+User support may first get in contact to ask if we hold data for a particular person. They may be identified by an email address, or another piece of data we hold on the system.
+
+When user support contact us we should:
+- Confirm with user support we are looking into it
+- Open a new trello ticket using the [access request template][3]
+- Assign a developer to confirm if we hold data on that user
+- Respond to user support confirming if we have data within 3 days.
+
+If we do hold data, user support may return to ask if we can export it for a user. They will agree a format, and scope.
```

Comment:
Does "they" refer to user support, the user, or both? General style point - minimise the use of they, it, and that in text to reduce possible ambiguity

Reply:
have gone through and removed they, much better thank you.